examples/dh-5-example/README.md

```diff
@@ -0,0 +1,27 @@
+This example helps Getting Started with using and configuring ingestion, mapping and mastering flows.
+There are 3 flow artifacts available in the flows directory.
+Before starting with this example, run the following gradle task to extract the input files directory from input.zip file
+./gradlew extractZip
+
+1) ingestion_only-flow.flow.json - This file contains 7 steps. Each step refers to ingesting different format of input files. The input files are located in the "input" directory
+
+2) ingestion_mapping-flow.flow.json - This file contains 4 steps. The input files are located in the "input" directory
+	Step 1 and Step 2 to ingest and harmonize (using mapping) json documents.
+	Step 3 and Step 4 to ingest and harmonize (using mapping) xml documents.
+
+3) ingestion_mapping_mastering-flow.flow.json - This file contains 3 steps. The input files are located in the "mastering-input" directory
+	Step 1 to ingest json documents
+	Step 2 to harmonize json documents
+	Step 3 to master json documents
+
+
+gradle task to run a Flow:
+
+1) ./gradlew hubRunFlow -PflowName=ingestion_only-flow
+2) ./gradlew hubRunFlow -PflowName=ingestion_mapping-flow
+3) ./gradlew hubRunFlow -PflowName=ingestion_mapping_mastering-flow

```

Comment:
This flow finished with a job status of "finished_with_errors". Here's a snippet of the error:

```
 "2": {
            "stepOutput": [
                "{\"stack\":\"Error: Could not find mapping: OrderMapping1 with version #1\\n    at Error (native)\\n    at main (/data-hub/5/builtins/steps/mapping/default/main.sjs:55:11)\\n    at Flow.runMain (/data-hub/5/impl/flow.sjs:374:12)\\n    at Flow.runStep (/data-hub/5/impl/flow.sjs:319:60)\\n    at Flow.runFlow (/data-hub/5/impl/flow.sjs:202:12)\\n    at post (/data-hub/5/extensions/runFlow.sjs:69:25)\\n    at callExtension (/data-hub/5/rest-api/lib/extensions-util.sjs:94:30)\\n    at applyOnce (/data-hub/5/rest-api/lib/extensions-util.sjs:112:10)\",\"message\":\"Could not find mapping: OrderMapping1 with version #1\",\"name\":\"Error\"}",
```

I tried running mlDeploy and mlLoadModules in case something wasn't loaded, but no luck. 
```

Reply:
Addressed the issue